### ISSUE TITLE
[1.3.3-rc.1] ecmascript "cannot find module es2015"

### ISSUE THREAD

**Issue by aldeed** [meteor/meteor#7179]
@benjamn, I updated a project to the just-released `1.3.3-rc.1` from the last `beta` and now I am getting this error on startup (happens with `rc.0` too):

```
While processing files with ecmascript (for target web.browser):
   /tools/isobuild/compiler-plugin.js:883:23: Cannot find module 'es2015'
```

I tracked down the cause of the error. We have a private package being pulled into the `packages` folder using the `mgp` tool. That package has a `.babelrc` file:

```
{
  "presets": ["es2015"]
}
```

The app itself does NOT have a `.babelrc` file.

The `.babelrc` in the package directory is used to build that package in its own repo, before it gets copied in locally to this app. So it should be ignored. Is there any reason in particular why Meteor is looking for `.babelrc` files outside of the app root?

A second point of weirdness: Normally to get the `es2015` preset to work in a pure Node app, we'd just `npm i --save-dev babel-preset-es2015`. But doing that in the Meteor app does not fix the error. I'm not sure how to get the compiler to "see" the "es2015" preset files.

*EDIT:* For anyone else having this problem, the workaround is to delete the package `.babelrc` file.

**Comment by benjamn** [comment]
I'm guessing the private package uses `ecmascript`?

I'm tempted to ignore or warn about `babel-preset-es2015` since it's so redundant with `babel-preset-meteor`. Perhaps warning (but not crashing) when a `.babelrc` dependency cannot be found would make sense in general, at least for package code. If the absence of a preset or plugin does not prevent your code from compiling, then maybe it wasn't necessary in the first place? What do you think about that?

If you have control over the private package's `package.js` file, you should be able to install npm dependencies like `babel-preset-es2015` via `Npm.depends`, if you really need to get it to work.

This should only be a problem for local packages in a `packages/` directory, by the way. Packages published to Atmosphere run compiler plugins (and examine `.babelrc` files) before publishing.

**Comment by aldeed** [comment]
OK, thanks for the explanation. Yes it uses ecmascript. Changing this to a warning makes sense to me since you will still have a clue as to why your project crashes later if it does crash.

**Comment by benjamn** [comment]
This will be fixed once https://github.com/meteor/meteor/pull/7033/commits/5ae53de46b3f2cc8331de54c7b254f40059a85b5 is released.

**Comment by scheung38** [comment]
So the 1.3.3-rc4 has a bug?

**Comment by benjamn** [comment]
@scheung38 no, that commit is included in `1.3.3-rc.4`

**Comment by scheung38** [comment]
Hi benjamn, 

I had a perfectly working app in 1.3.2.4, then after the meteor update --release 1.3.3-rc4 I have tons of errrors and breakages.. :
